src/components/TableOfContents/TableOfContents.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react"
+import { Stack } from "@chakra-ui/react"
+import { Meta, StoryObj } from "@storybook/react"
+
+import { ToCItem, TocNodeType } from "@/lib/types"
+
+import TableOfContents from "./"
+
+const tocItems: ToCItem[] = [
+  {
+    title: "The early Web",
+    url: "#early-internet",
+    items: [
+      { title: "Web 1.0: Read-Only (1990-2004)", url: "#web1" },
+      { title: "Web 2.0: Read-Write (2004-now)", url: "#web2" },
+    ],
+  },
+  {
+    title: "Web 3.0: Read-Write-Own",
+    url: "#web3",
+    items: [
+      {
+        title: "What is Web3?",
+        url: "#what-is-web3",
+        items: [{ title: "Core ideas of Web3", url: "#core-ideas" }],
+      },
+      {
+        title: "Why is Web3 important?",
+        url: "#why-is-web3-important",
+        items: [
+          { title: "Ownership", url: "#ownership" },
+          {
+            title: "Censorship resistance",
+            url: "#censorship-resistance",
+          },
+          {
+            title: "Decentralized autonomous organizations (DAOs)",
+            url: "#daos",
+          },
+        ],
+      },
+      { title: "Identity", url: "#identity" },
+      {
+        title: "Native payments",
+        url: "#native-payments",
+        items: [
+          { title: "Cryptocurrency", url: "#cryptocurrency" },
+          { title: "Micropayments", url: "#micropayments" },
+          { title: "Tokenization", url: "#tokenization" },
+        ],
+      },
+    ],
+  },
+  {
+    title: "Web3 limitations",
+    url: "#web3-limitations",
+    items: [
+      { title: "Accessibility", url: "#accessibility" },
+      { title: "User experience", url: "#user-experience" },
+      { title: "Education", url: "#education" },
+      {
+        title: "Centralized infrastructure",
+        url: "#centralized-infrastructure",
+      },
+    ],
+  },
+  {
+    title: "A decentralized future",
+    url: "#decentralized-future",
+  },
+  { title: "How can I get involved", url: "#get-involved" },
+  { title: "Further reading", url: "#further-reading" },
+]
+
+const meta = {
+  title: "Molecules / Navigation / TableOfContents",
+  parameters: {
+    layout: "fullscreen",
+  },
+  decorators: [
+    (Story) => (
+      <Stack minH="100vh" position="relative">
+        <Story />
+      </Stack>
+    ),
+  ],
+} satisfies Meta<typeof TableOfContents>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  render: () => <TableOfContents slug="#web3" items={tocItems} maxDepth={2} />,
```

Comment:
```log
9:11:34 AM: Failed to compile.
9:11:34 AM: 
9:11:34 AM: ./src/components/TableOfContents/TableOfContents.stories.tsx:94:34
9:11:34 AM: Type error: Type '{ slug: string; items: ToCItem[]; maxDepth: number; }' is not assignable to type 'IntrinsicAttributes & BoxProps & { items: ToCItem[]; maxDepth?: number | undefined; editPath?: string | undefined; hideEditButton?: boolean | undefined; isMobile?: boolean | undefined; }'.
9:11:34 AM:   Property 'slug' does not exist on type 'IntrinsicAttributes & BoxProps & { items: ToCItem[]; maxDepth?: number | undefined; editPath?: string | undefined; hideEditButton?: boolean | undefined; isMobile?: boolean | undefined; }'.
9:11:34 AM:   92 |
9:11:34 AM:   93 | export const Default: Story = {
9:11:34 AM: > 94 |   render: () => <TableOfContents slug="#web3" items={tocItems} maxDepth={2} />,
9:11:34 AM:      |                                  ^
9:11:34 AM:   95 | }
9:11:34 AM:   96 |
9:11:34 AM: error Command failed with exit code 1. (https://ntl.fyi/exit-code-1)
```

cc: @pettinarip Getting an error here... `slug` is not a prop on `TableOfContents`, do we need this here?

```tsx
// src/components/TableOfContents/index.tsx
export type TableOfContentsProps = BoxProps & {
  items: Array<ToCItem>
  maxDepth?: number
  editPath?: string
  hideEditButton?: boolean
  isMobile?: boolean
}
```